docs/whl.md

```diff
@@ -8,6 +8,22 @@
 whl_library(<a href="#whl_library-name">name</a>, <a href="#whl_library-extras">extras</a>, <a href="#whl_library-requirements">requirements</a>, <a href="#whl_library-whl">whl</a>)
 </pre>
 
+A rule for importing `.whl` dependencies into Bazel.
+
+<b>This rule is currently used to implement `pip_import`. It is not intended to
```

Comment:
Is this the right visual? What happens with    `stuff` when you are already in ```?

Reply:
It's not in a markdown triple-backtick, it's in a Python triple quote string (in source code, not the md file).

Reply:
You can preview the md file in github -- top right ellipses -> View file.